pkg/storage/internal/actions/funder/fee_calculator.go

```diff
@@ -0,0 +1,48 @@
+package funder
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/4chain-ag/go-wallet-toolbox/pkg/defs"
+	"github.com/go-softwarelab/common/pkg/to"
+)
+
+type feeCalc struct {
+	bytes float64
+	value uint64
+}
+
+func newFeeCalculator(model defs.FeeModel) *feeCalc {
+	if model.Type != defs.SatPerKB {
+		panic("unsupported fee model")
+	}
+
+	feeValue, err := to.UInt64(model.Value)
+	if err != nil {
+		panic("invalid fee model value: " + err.Error())
+	}
+	return &feeCalc{
+		value: feeValue,
+		bytes: 1000,
+	}
+}
+
+func (f *feeCalc) Calculate(txSize uint64) (uint64, error) {
+	size, err := to.Float64FromUnsigned(txSize)
+	if err != nil {
+		return 0, fmt.Errorf("invalid transaction size: %s", err.Error())
```

Comment:
```suggestion
		return 0, fmt.Errorf("invalid transaction size: %s", err)
```

Shouldn't it be like this here?

Reply:
my bad, it should be %w